### ISSUE TITLE
Dgraph Tour unable to connect

### ISSUE THREAD

**Issue by thandy1212** [dgraph-io/dgraph#1690]
Going through the dgraph tour I was able to successfully install docker and get dgraph up and running locally on docker through pulling the dgraph/dgraph image. I was able to access the database locally through my browser, however on the third page of the tour where it asks you to upload some sample data by pressing the run button it returns an error saying there is no connection. When I manually copy and paste the data in the browser dgraph console dgraph successfully adds the data.

**Comment by StarpTech** [comment]
Same here all examples doesn't works.

**Comment by peterstace** [comment]
I'm unable to reproduce the problem. @janardhan1993 , @pawanrawal can you guys try this out? It's important to get this issue sorted out quickly.

**Comment by pawanrawal** [comment]
Works well for me too. @thandy1212 @StarpTech are you guys using Firefox? Dgraph tour doesn't work with Firefox versions less than 55. You can read more about it at https://github.com/dgraph-io/dgraph/issues/1313#issuecomment-320591731.

**Comment by thandy1212** [comment]
I was using the new Firefox dev edition version 57.

**Comment by thandy1212** [comment]
Just tried it on Chrome as well and got same result

**Comment by thandy1212** [comment]
Here are the request headers as well as response header:

POST /query?latency=true HTTP/1.1
Host: 127.0.0.1:8080
Connection: keep-alive
Content-Length: 1413
Accept: application/json, text/javascript, */*; q=0.01
Origin: https://tour.dgraph.io
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36
Content-Type: application/x-www-form-urlencoded; charset=UTF-8
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.8

HTTP/1.1 200 OK
Access-Control-Allow-Credentials: true
Access-Control-Allow-Headers: Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token,X-Auth-Token, Cache-Control, X-Requested-With
Access-Control-Allow-Methods: POST, OPTIONS
Access-Control-Allow-Origin: *
Connection: close
Content-Type: application/json
Date: Mon, 30 Oct 2017 00:30:47 GMT
Content-Length: 74

Response Body
{"errors":[{"code":"Error","message":"No connection exists"}],"data":null}

In the console there were no error messages either

**Comment by thandy1212** [comment]
I got it working, the issue was that I had another instance of dgraph running that was running locally through my mac rather than through docker. My solution was to kill that process, delete the dgraph folder as well as the dgraph container. After cleaning everything up I reran the directions on page 2 of the tour. Once dgraph was running again I pressed the run button on page 3 of the tour and it responded with a success
